docs/_docs/localization.md

```diff
@@ -90,6 +97,27 @@ to the node `Hello,` which you can rename in this step.
 
 The plugin also allows you to set a proper description and char limit for translations.
 
+#### 1.4.1 Break identical strings into different translations
+
+Below is an exmaple of the output strings with the group option checked.
+
+<img src="./localization/GroupedExample.png">
+
+The strings grouping several text nodes will have a break icon button for each of the node, by clicking on which,
+it will create a separate entry for that node with a different string resource id. The string resource id is auto
+generated by the plugin. You can rename it. Note, if you rename it to an exising string resource name with the
+same text, it will merge them together.
```

Comment:
Hard to suggest over multiple lines, so:
"The strings grouping several text nodes will have a break icon button for each of node. By clicking this icon,
a separate entry for that node will be created with a different string resource id. The string resource id is auto
generated by the plugin, but you can rename it. Note, if you rename it to an existing string resource name with the same text, it will merge them together."

Reply:
Done.